vsintegration/src/FSharp.Editor/Hints/RoslynAdapter.fs

```diff
@@ -33,7 +33,9 @@ type internal RoslynAdapter [<ImportingConstructor>] (settings: EditorOptions) =
                     let! nativeHints = HintService.getHintsForDocument sourceText document hintKinds userOpName cancellationToken
 
                     let roslynHints =
-                        nativeHints |> Seq.map (NativeToRoslynHintConverter.convert sourceText)
+                        nativeHints
+                        |> Seq.filter (fun hint -> not hint.Parts.IsEmpty)
```

Comment:
Technically shouldn't be happening, would leave it as it was as it can help discover some bugs.

Reply:
Done